test/data/test_builtin_datasets.py

```diff
@@ -84,14 +89,29 @@ def test_penntreebank(self):
         self.assertEqual(len(train_dataset), 924412)
         self.assertEqual(len(test_dataset), 82114)
         self.assertEqual(len(valid_dataset), 73339)
+        self.assertEqual(train_dataset[20:25], torch.tensor([9919, 9920, 9921, 9922, 9188]).long())
+        self.assertEqual(test_dataset[30:35], torch.tensor([397, 93, 4, 16, 7]).long())
+        self.assertEqual(valid_dataset[40:45], torch.tensor([0, 0, 78, 426, 196]).long())
 
         vocab = train_dataset.get_vocab()
         tokens_ids = [vocab[token] for token in 'the player characters rest'.split()]
         self.assertEqual(tokens_ids, [2, 2550, 3344, 1125])
 
+        # Add test for the subset of the standard datasets
+        train_dataset, test_dataset = PennTreebank(data_select=('train', 'test'))
+        self.assertEqual(len(train_dataset), 924412)
+        self.assertEqual(len(test_dataset), 82114)
+        self.assertEqual(train_dataset[20:25], torch.tensor([9919, 9920, 9921, 9922, 9188]).long())
```

Comment:
nit: [torch.tensor](https://pytorch.org/docs/master/generated/torch.tensor.html#torch.tensor) has a dtype flag, so you can write ```torch.tensor([...], dtype=torch.int64)```.